lib/datadog/appsec/waf.rb

```diff
@@ -630,7 +630,7 @@ def run(persistent_data, ephemeral_data, timeout = LibDDWAF::DDWAF_RUN_TIMEOUT)
           valid!
 
           persistent_data_obj = Datadog::AppSec::WAF.ruby_to_object(
-            persistent_data,
+            persistent_data || {},
```

Comment:
Not sure about that? I can see the reason to default in ephemeral data, what is the reason here?